cs/src/core/FASTER.core.csproj

```diff
@@ -27,13 +27,11 @@
   <PropertyGroup Condition="'$(Configuration)' == 'Debug'">
     <DefineConstants>DEBUG</DefineConstants>
     <DebugType>full</DebugType>
-    <OutputPath>bin\$(Platform)\Debug\</OutputPath>
   </PropertyGroup>
```

Comment:
This still leaves
    src\core\bin\AnyCPU\Debug\net7.0\FASTER.core.xml
(which is also in src\core\bin\Debug\net7.0). This is controlled by 
<DocumentationFile>bin\$(Platform)\$(Configuration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
a few lines above (can't insert a comment there)